src/formElementPlugins/longText/pluginConfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "isCore": true,
+  "code": "longText",
+  "displayName": "Multi-line Text Input",
+  "dynamicParameters": ["label", "description", "placeholder"],
```

Comment:
```suggestion
  "dynamicParameters": ["label", "description", "placeholder", "lines"],
```

Reply:
Do you really think "lines" needs to be dynamic? I'm struggling to think of a use case where the number of lines available in a text box would be conditional on something else.